cmd/otel-agent/subcommands/status/command.go

```diff
@@ -0,0 +1,73 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+// Package status implements the core status component information provider interface
+package status
+
+import (
+	"context"
+	"os"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
+	agentConfig "github.com/DataDog/datadog-agent/cmd/otel-agent/config"
+	"github.com/DataDog/datadog-agent/cmd/otel-agent/subcommands"
+	coreconfig "github.com/DataDog/datadog-agent/comp/core/config"
+	status "github.com/DataDog/datadog-agent/comp/otelcol/status/def"
+	otelagentStatusfx "github.com/DataDog/datadog-agent/comp/otelcol/status/fx"
+	pkgconfigenv "github.com/DataDog/datadog-agent/pkg/config/env"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+type dependencies struct {
+	fx.In
+
+	Status status.Component
+}
+
+const headerText = "==========\nOTel Agent\n==========\n"
+
+// MakeCommand returns a `version` command to be used by agent binaries.
+func MakeCommand(globalConfGetter func() *subcommands.GlobalParams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Print the current status",
+		Long:  ``,
+		RunE: func(*cobra.Command, []string) error {
+			globalParams := globalConfGetter()
+			acfg, err := agentConfig.NewConfigComponent(context.Background(), globalParams.CoreConfPath, globalParams.ConfPaths)
+			if err != nil && err != agentConfig.ErrNoDDExporter {
+				return err
+			}
+			uris := append(globalParams.ConfPaths, globalParams.Sets...)
```

Comment:
It appears like the uris are a merge of `globalParams.ConfPaths` and `globalParams.Sets` here, whereas in the call to NewConfigComponent we only use `globalParams.ConfPaths` as the URIs, is this intended ? Can both contain collector config ?

Reply:
I did some more testing and I don't think this part is necessary for the status subcommand. I just updated my PR and based the fx portion off of https://github.com/DataDog/datadog-agent/blob/cf041df845d4a39ccf1fca09acc89d11aa7c7e3f/cmd/trace-agent/subcommands/info/command.go#L41-L54.

I confirmed this works without having to pass in config files since `coreconfig.NewAgentParams` falls back on a default config path (this is why `agent status` doesn't require a config to be passed in).